Client/sdk/game/CObject.h

```diff
@@ -41,4 +41,8 @@ class CObject : public virtual CPhysical
     virtual void     SetScale(float fX, float fY, float fZ) = 0;
     virtual CVector* GetScale() = 0;
     virtual void     ResetScale() = 0;
+
+    virtual bool IsOnFire() = 0;
```

Comment:
`const`?

Reply:
No, GetInterface in CEntitySA isn't const, and I will not edit additional classes

Reply:
oh wait haha I already asked about this. Sorry, my bad